src/game/Render/Render.cpp

```diff
@@ -77,3 +77,31 @@ bool R_PushRemoteScreenUpdate(std::int32_t a1)
     }
     return IsMainThread;
 }
+
+//DONE : 0x004CFA00
+std::int32_t FixWindowsDesktop()
+{
+    unsigned long currentThreadId = GetCurrentThreadId();
+    HWND desktopWindow = GetDesktopWindow();
+    HDC DC = GetDC(desktopWindow);
+    std::int32_t v3 = 0;
+    std::int8_t* v4;
+    std::int32_t v5 = 256;
+    void* ramp; //char[512]..? never used according to ida
```

Comment:
this is oddly defined, no clue if this is psuedocode going nuts
`unsigned __int16 ramp[3][256];`

and `v4` is defined as `unsigned __int16*`, and its `v4 = ramp[1]`? idk its a bunch of nonsense

Reply:
Yeah, I have no idea either, certified IDA moment.